src/components/drawer/drawer.js

```diff
@@ -36,6 +54,16 @@ export class DrawerComponent {
     );
   }
 
+  onMoreBtnClick(e) {
+    let all = this.mountPoint.querySelectorAll(".drawer__list-item");
```

Comment:
why do you need this code? what it does? I tried and it worked without this. Please recheck